src/nav/IconUnreadConversations.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
```

Comment:
You can now use the `ComponentWithOverlay.js` (in `/common/`) to achieve this without a creating a separate component.